.github/workflows/test.yml

```diff
@@ -41,14 +41,14 @@ jobs:
   docs:
     name: Docs
     runs-on: solang-ubuntu-latest
-    container: ubuntu:20.04
+    container: ubuntu:22.04
     steps:
       - name: Checkout sources
         uses: actions/checkout@v3
       - name: Install Python and git
         run: |
           apt-get update
-          apt-get install -y python3-pip git
+          apt-get install -y python3-pip git pkg-config libcairo-dev
```

Comment:
Changes to runners should ideally go in separate PRs, as they are not explained in the PR title nor in the description.

Reply:
So we have to weigh up how clean the history is vs effort needed to get it into that clean form. I am not sure it really is worth the effort to do this. If github provided linked PRs it would be much easier. Without that - and what we have at the moment, I would have to create one PR, get that merged and reviewed - for a silly change in dependencies in CI - before I could create the PRs of the actual work I wanted to do. Is that really worth it?

I used to think clean git history is very important, but github doesn't make that easy. Good work is separated into multiple commits which get squashed by github merging; you can't have dependencies between PRs; there is no good relation between git commit message and PR description (unless you squash merge, in which case the git commit messages gets discarded).

I don't think it's worth it, to be honest.

Reply:
This is one way to see it and perhaps for the problem you had another PR wasn't really necessary.

My concern was if we had multiple PRs waiting go to in and needed reviews. We don't know which will be merged first, but all of them need the CI fix. In this case, opening a separate PR would benefit all PRs waiting for review/merge.